hadoop-hdds/client/src/main/java/org/apache/hadoop/hdds/scm/storage/BufferPool.java

```diff
@@ -95,6 +95,17 @@ void releaseBuffer(ChunkBuffer chunkBuffer) {
     currentBufferIndex--;
   }
 
+  void releaseCurrentBuffer(ChunkBuffer chunkBuffer) {
```

Comment:
Is this method required? Seems to be unused and the same as the `releaseBuffer`, but I may miss sg.

Reply:
I cleared most of the unnecessary parts from BOS. I cleaned it. Thanks for the comment.